src/actix/mod.rs

```diff
@@ -98,13 +99,25 @@ pub fn init(
                 .service(scroll_points)
                 .service(count_points)
         })
-        .workers(max_web_workers(&settings))
-        .bind(format!(
-            "{}:{}",
-            settings.service.host, settings.service.http_port
-        ))?
-        .run()
-        .await
+        .workers(max_web_workers(&settings));
+
+        let bind_addr = format!("{}:{}", settings.service.host, settings.service.http_port);
+
+        let tls_settings = &settings.tls_config;
+
+        server = if settings.service.enable_tls {
+            let mut acceptor = SslAcceptor::mozilla_intermediate(SslMethod::tls())?;
```

Comment:
I guess, we can use [`SslAcceptor::mozilla_modern_v5`][ssl-acceptor-mozilla-modern-v5] (or, at least, [`SslAcceptor::mozilla_modern`][ssl-acceptor-mozilla-modern]) here.

[`SslAcceptor::mozilla_modern_v5`][ssl-acceptor-mozilla-modern-v5] requires OpenSSL 1.1.1... [which has been released in 2018 (scroll to the bottom)][openssl-release-notes]. Seems reasonable.

```suggestion
            let mut acceptor = SslAcceptor::mozilla_modern_v5(SslMethod::tls())?;
```

[ssl-acceptor-mozilla-modern-v5]: https://docs.rs/openssl/latest/openssl/ssl/struct.SslAcceptor.html#method.mozilla_modern_v5
[ssl-acceptor-mozilla-modern]: https://docs.rs/openssl/latest/openssl/ssl/struct.SslAcceptor.html#method.mozilla_modern
[openssl-release-notes]: https://www.openssl.org/news/openssl-1.1.1-notes.html

Reply:
Changed it. Let's hope there won't be need to change it back :)